docs/reference/commandline/login.md

```diff
@@ -97,6 +98,16 @@ For example, to use `docker-credential-osxkeychain`:
 If you are currently logged in, run `docker logout` to remove
 the credentials from the file and run `docker login` again.
 
+### Default behavior
+
+By default, Docker looks for the native binary on each of the platforms, i.e.
+"osxkeychain" on macOS, "wincred" on windows, and "pass" on Linux. A special
+case is that on Linux, Docker will fall back to the "secretservice" binary if
+it cannot find the "pass" binary.
+
+*Note:* If you do not supply a credential helper binary, Docker stores your
```

Comment:
This should  be formatted as a note;

```markdown
> **Note**: If you do not supply a credential helper binary, Docker stores your
> password in cleartext (base64 encoded) in your home directory!
```

Should this mention the actual file this is saved in as well (`~/.docker/config.json`)?

Reply:
I see this is being referred to a couple lines up as well;

> You can log into any public or private repository for which you have credentials. When you log in, the command stores encoded credentials in `$HOME/.docker/config.json` on Linux or `%USERPROFILE%/.docker/config.json` on Windows.